AuthenticationExample/AuthenticationExample/AuthenticationExample.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.app-sandbox</key>
+	<true/>
+	<key>com.apple.security.network.client</key>
```

Comment:
is this new entitlement necessary?

Reply:
It was automatically added by Xcode when I added the destination

Reply:
It is there for Example app as well
https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/main/Examples/Examples.xcodeproj/project.pbxproj#L60